test/plugins/graphql.spec.js

```diff
@@ -156,35 +165,63 @@ describe('Plugin', () => {
       })
 
       it('should instrument operations', done => {
-        const source = `query MyQuery { hello(name: "world") }`
+        const source = `query MyQuery($who: String!) { hello(name: $who) }`
 
         agent
           .use(traces => {
-            const spans = sort(traces[0])
+            const span = findSpan(traces[0], 'query', 'query MyQuery')
+            expect(span.meta).to.have.property('graphql.document', source)
+            expect(span.meta).to.have.property('graphql.variables', JSON.stringify({ who: 'world' }))
+          })
+          .then(done)
+          .catch(done)
 
-            expect(spans).to.have.length(3)
```

Comment:
I prefer to keep these as historically there have been bugs found from those checks.

Reply:
Ok 👍